pkg/controllers/infracluster/infracluster_controller_test.go

```diff
@@ -0,0 +1,266 @@
+package infracluster
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-capi-operator/pkg/operatorstatus"
+	"github.com/openshift/cluster-capi-operator/pkg/test"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	awsv1 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var _ = Describe("InfraCluster", func() {
+	ocpInfraAWS := configv1.Infrastructure{
+		Status: configv1.InfrastructureStatus{
+			APIServerInternalURL: "https://api-int.test-cluster.test-domain:6443",
+			Platform:             configv1.AWSPlatformType,
+			PlatformStatus: &configv1.PlatformStatus{
+				Type: configv1.AWSPlatformType,
+				AWS: &configv1.AWSPlatformStatus{
+					Region: "us-east-1",
+				},
+			},
+		},
+	}
+
+	infraClusterWithExternallyManagedByAnnotation := &awsv1.AWSCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      defaultInfraClusterName,
+			Namespace: defaultCAPINamespace,
+			Annotations: map[string]string{
+				clusterv1.ManagedByAnnotation: managedByAnnotationValueClusterCAPIOperator,
+			},
+		},
+	}
+
+	// When the annotation is present and the value is empty, it is CAPI owned. When the annotation is
```

Comment:
Why empty for us owning it, shouldn't we have a unique value?

Reply:
Up until now, we only set the key of the `clusterv1.ManagedByAnnotation`. So this is consistent with the existing approach.
We could change this of course, but then there might be upgrade consequences. WDYT?

Reply:
Upgrade for? We don't support anything in CAPI right now right? So changing it, yeah we would have to change our docs etc, but, wouldn't/shouldn't break anyone in production yet?